app.js

```diff
@@ -161,8 +165,21 @@ function toggleHeartDisplay(event) {
 
 function checkFaves(id) {
     const favorites = localStorage.getItem('favorites');
-    if (!favorites) {
+    if(!favorites) {
         return false;
     }
     return Boolean(favorites.split(',').find(elem => elem == id));
 }
+
+function paginationCheck(pageButton) {  
+    if(pageButton === 'next-page' && currentPageData.next) {
+        return true; 
+    } else if(pageButton === 'previous-page' && currentPageData.prev) {
+        return true;
+    } else {return false;}
+}
```

Comment:
I've given a refactoring suggestion for this function above. I'll give some context here.

If you ever have code of the form 
```js
if (myCondition) {
    return true
} else {
    return false
}
```
that's the same as doing this
```js
return myCondition;
```
**_as long as `myCondition` is actually a boolean._** 

---

In my suggested version of this function, we actually end up returning the value of `currentPageData.next` and `currentPageData.prev`. If we wanted to only ever return a pure boolean we could do something like:

```js
function paginationCheck(pageButton) {
    const pageData = pageButton === 'next-page' ? currentPageData.next : currentPageData.prev;
    return Boolean(pageData)
}
```

But since we only ever call this function inside of an `if` condition, I think it's fine to return truthy values instead of booleans.

Reply:
That makes sense -- I agree that since the truthiness of `pageData` is being checked, the returned value doesn't necessitate a boolean. Very sleek one-liner to boot 🤠